tiingo/src/adapter.ts

```diff
@@ -13,19 +13,12 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
 
   Requester.logConfig(config)
 
-  const jobRunID = validator.validated.id
   const endpoint = validator.validated.data.endpoint || DEFAULT_ENDPOINT
 
   switch (endpoint) {
-    case eod.NAME: {
-      return await eod.execute(request, config)
-    }
+    case eod.NAME:
     default: {
-      throw new AdapterError({
-        jobRunID,
-        message: `Endpoint ${endpoint} not supported.`,
-        statusCode: 400,
-      })
+      return await eod.execute(request, config)
```

Comment:
Just out of curiousity, was a different endpoint name being requested that caused this to get a default added?

Reply:
I believe they were using a single job spec across various adapters. Some adapters used `endpoint: stock` which caused the tiingo EA to throw an error. So now it defaults to the only endpoint no matter what the `endpoint` parameter is set to